plugins/es-analysis-common/src/test/java/io/crate/analysis/common/CommonAnalyzerITest.java

```diff
@@ -129,4 +131,74 @@ CREATE ANALYZER shingle_default (
         var impl = new DisableGraphAttributeImpl();
         assertThat(impl).isInstanceOf(DisableGraphAttribute.class);
     }
+
+    @Test
+    public void test_add_column_with_custom_analyzer() {
+        execute("""
+            CREATE ANALYZER comma_separation_analyzer EXTENDS "standard" with (
+                TOKENIZER mypattern WITH (
+                   type = 'pattern',
+                   pattern = ',\\\\s'
+                )
+                , TOKEN_FILTERS (lowercase)
+            )
+            """);
+        execute("CREATE TABLE tbl (keywords TEXT)");
+
+        execute("""
+            ALTER TABLE tbl
+            ADD COLUMN keywords_analyzed TEXT INDEX USING FULLTEXT WITH (analyzer = 'comma_separation_analyzer')
+            """);
+        execute("""
+            INSERT INTO tbl (keywords, keywords_analyzed) VALUES(
+            'some articles',
+            'Humans, Computational Biology, Reactive Oxygen Species, Superoxide Dismutase, Rare Diseases, Gene Ontology, Oxidative Stress')
+            """);
+        execute("REFRESH TABLE tbl");
+        execute("""
+            SELECT keywords FROM tbl
+            WHERE MATCH(keywords_analyzed, 'biology')
+            """);
+        assertThat(response).hasRows("some articles");
+        execute("DROP ANALYZER comma_separation_analyzer");
```

Comment:
I would move the drop into a teardown to ensure it always runs. Otherwise there's a chance that if this test fails due to some changes, the missing drop will cause other tests to fail too - making it harder to spot the actual issue.

Reply:
Unfortunately, we don't have a `DROP ANALYZER IF EXISTS`, and also I couldn't find a way to get from `information_schema.routines` only the custom analyzers and execute a DROP based on a query on that table.
I can just add a `try/finally` to make sure this runs at the end of the test.

Reply:
Or do it like in https://github.com/crate/crate/blob/738f3d3a694711ef1ac5d2651e9342d8dc7b27a8/server/src/test/java/io/crate/integrationtests/SnapshotRestoreIntegrationTest.java#L128-L139